velox/connectors/hive/storage_adapters/hdfs/HdfsWriteFile.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <hdfs/hdfs.h>
+#include <vector>
+#include "velox/common/file/File.h"
+
+namespace facebook::velox {
+
+class HdfsWriteFile : public WriteFile {
```

Comment:
Please, document this class and its methods. Make sure to use "override" as needed.